src/main/java/org/opentestsystem/delivery/logging/EventInfo.java

```diff
@@ -1,79 +1,93 @@
 package org.opentestsystem.delivery.logging;
 
-import com.google.auto.value.AutoValue;
 import com.google.common.base.Optional;
 import org.opentestsystem.delivery.logging.EventLogger.EventData;
 
+import java.util.HashMap;
 import java.util.Map;
 
-@AutoValue
+import static org.apache.commons.lang.StringUtils.trimToEmpty;
+
 /**
  * Event to be logged to central logging
  * Used to simplify the number of parameters sent to the logger
  */
-public abstract class EventInfo {
+public class EventInfo {
+  private String event;
+  private String checkpoint;
+  private String message;
+  private final Map<EventData, Object> data = new HashMap<>();
 
   /**
    * @return Name of the current event
    */
-  abstract String event();
+  public String event() {
+    return event;
+  }
+
+  void setEvent(final String event) {
+    this.event = event;
+  }
 
   /**
    * @return Moment in time of the event, such as the entry, exit or error
    */
-  abstract Optional<String> checkpoint();
+  public Optional<String> checkpoint() {
+    return Optional.fromNullable(checkpoint);
+  }
+
+  void setCheckpoint(final String checkpoint) {
+    this.checkpoint = checkpoint;
+  }
 
   /**
    * @return Optional message for the event
    */
-  abstract Optional<String> message();
+  public Optional<String> message() {
+    return Optional.fromNullable(message);
+  }
+
+  void setMessage(final String message) {
+    this.message = message;
+  }
 
   /**
    * @return Event data payload that is used to access the event from log files
    */
-  abstract Map<EventData, Object> data();
-
-  public static EventInfo create(String event, String checkpoint, String message, Map<EventData, Object> data) {
-    return builder()
-      .event(event)
-      .checkpoint(Optional.of(checkpoint))
-      .message(Optional.of(message))
-      .data(data)
-      .build();
+  public Map<EventData, Object> data() {
+    return data;
   }
 
-  public static EventInfo create(String event, String checkpoint, Map<EventData, Object> data) {
-    return builder()
-      .event(event)
-      .checkpoint(Optional.of(checkpoint))
-      .message(Optional.<String>absent())
-      .data(data)
-      .build();
+  void setData(final Map<EventData, Object> data) {
+    this.data.putAll(data);
   }
 
-  public static EventInfo create(String event, Map<EventData, Object> data) {
-    return builder()
-      .event(event)
-      .checkpoint(Optional.<String>absent())
-      .message(Optional.<String>absent())
-      .data(data)
-      .build();
+  /**
+   * Create a builder containing this EventInfo's values.
+   *
+   * @return A builder
+   */
+  public EventInfoBuilder toBuilder() {
+    return builder().copy(this);
   }
```

Comment:
Again, this feels like an unnecessary shortcut for `EventInfo.builder().copy(myObj)` but since it already existed... =)